### PR TITLE
en, zh: Add telemetry notes

### PR DESCRIPTION
**Pull request by breeswish**
### What is changed, added or deleted? (Required)

Fix https://github.com/pingcap/docs-tidb-operator/issues/509
Add notes about the telemetry for all deployment or usage path (Chinese doc).

TODO:

- [ ] Wait for https://github.com/pingcap/docs-cn/pull/3812 being merged
- [x] Add notes for EN doc

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

**Comment by breeswish** [comment]
TODO: Add notes for EN doc

File: zh/deploy-on-alibaba-cloud.md [hunk 1]
```diff
@@ -255,6 +255,10 @@ mysql -h ${tidb_lb_ip} -P 4000 -u root
 
 `tidb_lb_ip` 为 TiDB Service 的 LoadBalancer IP。
 
+> **注意：**
+>
+> TiDB (v4.0.2 版本起) 默认会定期收集使用情况信息，并将其分享给 PingCAP 用于改善产品。若要了解收集的信息详情及如何禁用这个行为，请参见[遥测](https://docs.pingcap.com/zh/tidb/stable/telemetry)章节。
```
Comment (ran-huang) [inline zh/deploy-on-alibaba-cloud.md:260]:
```suggestion
> TiDB（v4.0.2 起）默认会定期收集使用情况信息，并将其分享给 PingCAP 用于改善产品。若要了解收集的信息详情及如何禁用这个行为，请参见[遥测](https://docs.pingcap.com/zh/tidb/stable/telemetry)章节。
```

File: zh/deploy-on-aws-eks.md [hunk 2]
```diff
@@ -490,6 +490,10 @@ mysql -h ${tidb_lb} -P 4000 -u root
     helm ls
     ```
 
+> **注意：**
+>
+> TiDB (v4.0.2 版本起) 默认会定期收集使用情况信息，并将其分享给 PingCAP 用于改善产品。若要了解收集的信息详情及如何禁用这个行为，请参见[遥测](https://docs.pingcap.com/zh/tidb/stable/telemetry)章节。
```
Comment (ran-huang) [inline zh/deploy-on-aws-eks.md:495]:
```suggestion
> TiDB（v4.0.2 起）默认会定期收集使用情况信息，并将其分享给 PingCAP 用于改善产品。若要了解收集的信息详情及如何禁用这个行为，请参见[遥测](https://docs.pingcap.com/zh/tidb/stable/telemetry)章节。
```

File: zh/deploy-on-gcp-gke.md [hunk 3]
```diff
@@ -239,6 +239,10 @@ Terraform 自动加载和填充匹配 `terraform.tfvars` 或 `*.auto.tfvars` 文
     > 通过 MySQL 连接 TiDB 前，需要先安装 MySQL 客户端。CentOS 系统可通过 `sudo yum install -y mysql` 安装。
     > `${tidb_ilb_ip}` 为前面获取的 Internal LoadBalancer IP 地址。
 
+> **注意：**
+>
+> TiDB (v4.0.2 版本起) 默认会定期收集使用情况信息，并将其分享给 PingCAP 用于改善产品。若要了解收集的信息详情及如何禁用这个行为，请参见[遥测](https://docs.pingcap.com/zh/tidb/stable/telemetry)章节。
```
Comment (ran-huang) [inline zh/deploy-on-gcp-gke.md:244]:
```suggestion
> TiDB（v4.0.2 起）默认会定期收集使用情况信息，并将其分享给 PingCAP 用于改善产品。若要了解收集的信息详情及如何禁用这个行为，请参见[遥测](https://docs.pingcap.com/zh/tidb/stable/telemetry)章节。
```

File: zh/upgrade-a-tidb-cluster.md [hunk 4]
```diff
@@ -125,3 +125,7 @@ helm upgrade ${release_name} pingcap/tidb-cluster -f values.yaml --version=${cha
 > ```shell
 > kubectl annotate tc ${release_name} -n ${namespace} tidb.pingcap.com/force-upgrade-
 > ```
+
```
Comment (DanielZhangQD) [inline zh/upgrade-a-tidb-cluster.md:128]: Is it better we move this warning to the beginning of this doc, e.g. L16?

File: zh/deploy-on-general-kubernetes.md [hunk 5]
```diff
@@ -200,3 +200,7 @@ TiDB Operator 部署并配置完成后，可以通过下面命令部署 TiDB 集
 ### 初始化 TiDB 集群
 
 如果要在部署完 TiDB 集群后做一些初始化工作，参考 [Kubernetes 上的集群初始化配置](initialize-a-cluster.md)进行配置。
+
+> **注意：**
+>
+> TiDB (v4.0.2 版本起) 默认会定期收集使用情况信息，并将其分享给 PingCAP 用于改善产品。若要了解收集的信息详情及如何禁用这个行为，请参见[遥测](https://docs.pingcap.com/zh/tidb/stable/telemetry)章节。
```
Comment (ran-huang) [inline zh/deploy-on-general-kubernetes.md:206]:
```suggestion
> TiDB（v4.0.2 起）默认会定期收集使用情况信息，并将其分享给 PingCAP 用于改善产品。若要了解收集的信息详情及如何禁用这个行为，请参见[遥测](https://docs.pingcap.com/zh/tidb/stable/telemetry)章节。
```

File: zh/deploy-tidb-from-kubernetes-gke.md [hunk 6]
```diff
@@ -247,6 +247,10 @@ SET PASSWORD FOR 'root'@'%' = '<change-to-your-password>';
 
 恭喜，你已经启动并运行一个兼容 MySQL 的分布式 TiDB 数据库！
 
+> **注意：**
+>
+> TiDB (v4.0.2 版本起) 默认会定期收集使用情况信息，并将其分享给 PingCAP 用于改善产品。若要了解收集的信息详情及如何禁用这个行为，请参见[遥测](https://docs.pingcap.com/zh/tidb/stable/telemetry)章节。
```
Comment (ran-huang) [inline zh/deploy-tidb-from-kubernetes-gke.md:252]:
```suggestion
> TiDB（v4.0.2 起）默认会定期收集使用情况信息，并将其分享给 PingCAP 用于改善产品。若要了解收集的信息详情及如何禁用这个行为，请参见[遥测](https://docs.pingcap.com/zh/tidb/stable/telemetry)章节。
```

File: zh/deploy-tidb-from-kubernetes-kind.md [hunk 7]
```diff
@@ -184,6 +184,10 @@ kubectl -n pingcap get pods
     >
     > 然后，在浏览器中打开 `http://<远程主机 IP>:3000` 访问 Grafana 监控面板。
 
+> **注意：**
+>
+> TiDB (v4.0.2 版本起) 默认会定期收集使用情况信息，并将其分享给 PingCAP 用于改善产品。若要了解收集的信息详情及如何禁用这个行为，请参见[遥测](https://docs.pingcap.com/zh/tidb/stable/telemetry)章节。
```
Comment (ran-huang) [inline zh/deploy-tidb-from-kubernetes-kind.md:189]:
```suggestion
> TiDB（v4.0.2 起）默认会定期收集使用情况信息，并将其分享给 PingCAP 用于改善产品。若要了解收集的信息详情及如何禁用这个行为，请参见[遥测](https://docs.pingcap.com/zh/tidb/stable/telemetry)章节。
```

File: zh/deploy-tidb-from-kubernetes-minikube.md [hunk 8]
```diff
@@ -212,6 +212,10 @@ kubectl get svc -n demo --watch
     mysql -h 127.0.0.1 -P 4000 -uroot -e 'select tidb_version();'
     ```
 
+> **注意：**
+>
+> TiDB (v4.0.2 版本起) 默认会定期收集使用情况信息，并将其分享给 PingCAP 用于改善产品。若要了解收集的信息详情及如何禁用这个行为，请参见[遥测](https://docs.pingcap.com/zh/tidb/stable/telemetry)章节。
```
Comment (ran-huang) [inline zh/deploy-tidb-from-kubernetes-minikube.md:217]:
```suggestion
> TiDB（v4.0.2 起）默认会定期收集使用情况信息，并将其分享给 PingCAP 用于改善产品。若要了解收集的信息详情及如何禁用这个行为，请参见[遥测](https://docs.pingcap.com/zh/tidb/stable/telemetry)章节。
```

File: zh/upgrade-a-tidb-cluster.md [hunk 9]
```diff
@@ -125,3 +125,7 @@ helm upgrade ${release_name} pingcap/tidb-cluster -f values.yaml --version=${cha
...
+> **注意：**
+>
+> TiDB (v4.0.2 版本起) 默认会定期收集使用情况信息，并将其分享给 PingCAP 用于改善产品。若要了解收集的信息详情及如何禁用这个行为，请参见[遥测](https://docs.pingcap.com/zh/tidb/stable/telemetry)章节。
```
Comment (ran-huang) [inline zh/upgrade-a-tidb-cluster.md:131]:
```suggestion
> TiDB（v4.0.2 起）默认会定期收集使用情况信息，并将其分享给 PingCAP 用于改善产品。若要了解收集的信息详情及如何禁用这个行为，请参见[遥测](https://docs.pingcap.com/zh/tidb/stable/telemetry)章节。
```

File: en/deploy-on-gcp-gke.md [hunk 10]
```diff
@@ -242,6 +242,10 @@ After `terraform apply` is successful executed, perform the following steps to a
     > You need to install the MySQL client before you connect to TiDB via MySQL. If you use CentOS, install the client by executing `sudo yum install -y mysql`.
     > `${tidb_ilb_ip} is the IP address of the Internal LoadBalancer acquired in step 1.
 
+> **Note:**
```
Comment (ran-huang) [inline en/deploy-on-gcp-gke.md:245]: Could you put this **Note** somewhere else if possible? It looks weird that two notes are so close to each other.

File: zh/upgrade-a-tidb-cluster.md (hunk 4 above)
Comment (breeswish) [reply zh/upgrade-a-tidb-cluster.md:128]: It's a note that user can know but will not cause harm when user is not aware of.

Comment (breeswish) [reply zh/upgrade-a-tidb-cluster.md:128]: We are also following [CRDB's way](https://www.cockroachlabs.com/docs/stable/install-cockroachdb-mac.html) to put this kind of friendly note at a secondary place.

File: en/deploy-on-gcp-gke.md (hunk 10 above)
Comment (breeswish) [reply en/deploy-on-gcp-gke.md:245]: Good catch, I will find a new place to put it.

**Pull request merged**